app/1.0/docs/devguide/local-dom.md

```diff
@@ -18,6 +18,25 @@ when manipulating DOM from JavaScript. This interface covers most of the
 common DOM methods and properties, and is compatible with both shady DOM and
 native shadow DOM.
 
+```html
+<dom-module>
+  <template>
+  
+    <!-- 
+      Local DOM. Possible implementations : 
+        - Shadow DOM (browser implementation)
+        - Shady DOM (Polymer implementation)
+    -->
+    
+    <content>
```

Comment:
so we should keep it `<content>` even though 1.7+ can make use of slots and reduces changes going to 2.0?